### ISSUE TITLE
Kapitel 6 Identifizierungskennzeichen

### ISSUE THREAD

**Issue by mme1950** [ASJeton/Klassenvorschrift#2]
In diesem Kapitel existieren mehrere nicht essentielle Regeln, die bei vielen Booten verletzt sind:

6.2 verlangt, dass die Segelnummer in der oberen Hälfte des Segels aufgeklebt ist. Wegen der Latten und "GER" statt "G" passt das nicht. Ich denke, das kann man ohne weiteres ändern.

6.3 verlangt, dass die Segelnummern rot sein müssen. Das hätte eingehalten werden können. Aber müssen wir die Regel wirklich haben? 

In Absatz 6 Identifizierungskennzeichen werden folgende Änderungen vorgenommen:
* Das Nationalitätskennzeichen und die Segelnummer muss jetzt oberhalb des Drittels der Höhe des Großsegels angebracht werden.
* Die Farbe der Segelnummer wird freigestellt, muss jedoch kontrastierend sein.
* Absätze werden anders geschnitten:
  * 6.2: Positionierung und G vs. GER.
  * 6.3: Ausführung der Zeichen und Schriften.
* Reihenfolge immer Klassenzeichen, Nationalitätskennzeichen, Segelnummer.
* Einheitliche Begriffe: Klassenzeichen, Nationalitätskennzeichen, Segelnummer.

**Comment by mme1950** [comment]
Absatz 6.2 (bisher):
> Im Großsegel muß sich oberhalb der halben Höhe die Segelnummer befinden. Das Klassenzeichen ist im Großsegel über der Segelnummer anzubringen. Im Großsegel ist zusätzlich das Nationalitätszeichen vor oder über der Segelnummer zu führen. Als Nationalitätskennzeichen müssen entweder die vor dem 31. März 1999 vorgeschriebenen oder die danach in WR 77 vorgeschriebenen Nationalitätsbuchstaben gefahren werden.

Absatz 6.2 schreibe neu:
> Im Großsegel sind das Klassenzeichen und darunter das Nationalitätskennzeichen und die Segelnummer anzubringen. Das Nationalitätskennzeichen steht vor oder über Segelnummer. Die Segelnummer muss sich oberhalb eines Drittels der Höhe des Großsegels befinden. Nationalitätskennzeichen und Segelnummern sind auf beiden Seiten des Großsegels in unterschiedlicher Höhe anzubringen, an Steuerbord höher als an Backbord.

**Comment by mme1950** [comment]
Absatz 6.3 (bisher):
> Die Segelnummer steht unter dem Klassenzeichen und muß **300±20 mm** hoch sein, die Schriftstärke beträgt mindestens **50 mm**. 
> Klassenzeichen und Segelnummer müssen in roter Farbe ausgeführt sein und sind auf beiden Seiten des Großsegels in unterschiedlicher Höhe anzubringen, an Steuerbord höher als an Backbord. Das Segelzeichen des Jeton besteht aus einem in der Mitte zweigeteilten Kreis; die obere Hälfte ist rot, die untere Hälfte enthält ein rotes “J”. Der Durchmesser des Segelzeichens beträgt **320±20 mm** (gemäß Segelblatt).

Absatz 6.3 schreibe neu:
> Das Klassenzeichen des Jeton besteht aus einem in der Mitte zweigeteilten Kreis; die obere Hälfte ist rot, die untere Hälfte enthält ein rotes “J”. Der Durchmesser des Segelzeichens beträgt  **320±20 mm** (gemäß Messblatt 3).
> Höhe der Buchstaben des Nationalitätskennzeichens und Zahlen der Segelnummer     **300±20 mm**
> Schriftstärke     **min. 50 mm**
> Als Nationalitätskennzeichen müssen entweder die vor dem 31. März 1999 vorgeschriebenen oder die danach in WR 77 vorgeschriebenen Nationalitätsbuchstaben gefahren werden.
> Das Nationalitätskennzeichen und die Segelnummer sind in kontrastierender Farbe anzubringen.

**Comment by daniel-richter** [comment]
Die neuformulierten Texte weichen im Satzbau bzw. der Reihenfolge der geregelten Sachen voneinander ab. Ist das Absicht? Unter Beibehaltung der alten Textstruktur würde man die tatsächlichen Änderungen besser erkennen.

Ist es Absicht, dass in Absatz 6.2 der Passus "Als Nationalitätskennzeichen müssen entweder die vor dem 31. März 1999 vorgeschriebenen oder die danach in WR 77 vorgeschriebenen Nationalitätsbuchstaben gefahren werden." in der Neuformulierung fehlt?

**Comment by mme1950** [comment]
Ich habe den Text bewusst anders geschnitten, siehe oben.
Der Passus fehlte noch, gehört aber in 6.3, finde ich. Das habe ich oben noch eingefügt.

**Comment by daniel-richter** [comment]
Apropos: 6.4 fordert, dass die Segelnummer auch im Spi sichtbar sein muss - was bei den wenigsten Booten der Fall ist.

> 6.4 Auf dem Spinnaker sind Nationalitätszeichen und Unterscheidungsnummer (Größe unter 6.3) in ähnlicher Weise jedoch in annähernd halber Höhe in kontrastierender Farbe anzubringen.

**Comment by mme1950** [comment]
Ja, da hast du recht. Keiner möchte auf seinen Spi eine Nummer kleben oder mit Edding drauf schreiben. Dann kann man den Spi nicht tauschen. Ich würde das aber in der Grauzone lassen. Das ist jetzt Jahrzehnte lang niemandem aufgestoßen.

BTW: Findest du, dass der Passus mit G <--> GER eher zu 6.2 oder zu 6.3 gehört? Wie ich jetzt sehen, war mir auch nicht sicher.

**Comment by daniel-richter** [comment]
> Ja, da hast du recht. Keiner möchte auf seinen Spi eine Nummer kleben oder mit Edding drauf schreiben. Dann kann man den Spi nicht tauschen. Ich würde das aber in der Grauzone lassen. Das ist jetzt Jahrzehnte lang niemandem aufgestoßen.

Naja, nach der Argumentation brauchen wir den Rest aber auch nicht zu ändern. Die Farbe und Position der Segelnummern im Großsegel ist jahrzentelang ebenso niemanden aufgestoßen, also warum das nicht auch in der Grauzone lassen?
(Meine Meinung: Wenn, dann richtig. Also auch um die Segelnummer im Spi kümmern.)

> BTW: Findest du, dass der Passus mit G <--> GER eher zu 6.2 oder zu 6.3 gehört? Wie ich jetzt sehen, war mir auch nicht sicher.

6.2 beschreibt mEn, _was_ zu sehen sein muss; 6.3, _wie_ es auszusehen hat. Vmtl. würde ich es daher bei 6.2 einordnen.

Apropos: Schön, dass in 6.3 die Schrift des Nationalitätszeichens mit definiert wird (war vorher ja nicht so).

Wording: Warum heißt es eigentlich "[a]ls Nationalitätskennzeichen müssen [...] die [...] Nationalitätsbuchstaben **gefahren** werden"? Als Verb hätte ich eher sowas wie "verwendet" erwartet.
